reflectable/pubspec.yaml

```diff
@@ -1,13 +1,13 @@
 name: reflectable
-version: 3.0.10
+version: 3.1.0
```

Comment:
Consider doing a major increment now that you have breaking changes.

Reply:
Yes! For quite a while I was working on this PR targeting analyzer 4.4.0 (and that one already contained the breaking change on the analyzer side about mixins), but analyzer 5.0.0 came out since then, and this means that we can piggyback on the new major version of the analyzer. So 4.0.0 it is.